tools/banner-studio/index.html

```
@@ -117,6 +124,7 @@
   <label>Colour</label><div class="row"><input type="color" id="bgPick" value="#1e2126" style="width:34px;height:28px;border:0;background:none"><button class="act" id="bgUse">Apply</button><button class="act" id="bgReset">Auto</button></div>
   <label>Background image (auto-dimmed for contrast)</label><input type="file" id="bgFile" accept="image/*">
   <div class="row" style="margin-top:6px"><button class="act" id="bgClear" style="flex:1">Remove image</button></div>
+  <label>Background blur <span id="bbV"></span></label><input type="range" id="bgBlur" min="0" max="20" step="0.5">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)